fah-client/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v2
```

Comment:
So this chart requires Helm 3? What about our current setups?

Reply:
Ah I see you already deployed it as a v3 chart on our own cluster :P

Reply:
yeah, given that we'll deploy this chart manually for now (not in addons stack), I figured it would be better to already do it via Helm v3